File: JewelsAndStones.cs
public class Solution {
    public int NumJewelsInStones(string jewels, string stones) {
        HashSet<char> skup = new HashSet<char>();
        
        for(int i=0; i<jewels.Length; i++) {
            skup.Add(jewels[i]);
        }
        int c = 0;
        for(int i=0; i<stones.Length; i++) {
            if(skup.Contains(stones[i])) {
                c++;
            }
        }
        return c;
    }
}

File: JewelsAndStones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        HashSet<Character> skup = new HashSet<>();
        for(int i=0; i<jewels.length(); i++) {
            skup.add(jewels.charAt(i));
        }
        
        int counter = 0;
        for(int i=0; i<stones.length(); i++) {
            if(skup.contains(stones.charAt(i))) {
                counter = counter + 1;
            }
        }
        return counter;
    }
}
